LICENSE.txt

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
+
+Copyright (c) 2007-2020, mlpack
```

Comment:
Hm, not sure we can use the same year here, because the models repo is somewhat new; and this reads like we out a 'new' license to an older version, but I'm not an expert.

Reply:
That's a good question. I think it's reasonable to say that all of mlpack is split from one project, and thus can share the 2007 copyright date, but you're right that it probably makes more sense to use a newer year for this repo.

Reply:
Following the date on https://github.com/mlpack/models/commit/2f547f3345ca308f43b6d6a2d17fcdf94677f59f
```suggestion
Copyright (c) 2017-2020, mlpack
```